available_services/irrigationservice.yml

```diff
@@ -0,0 +1,40 @@
+services:
+  irr_db:
+    image: postgres:16.3
+    environment:
+      POSTGRES_DB: ${IRR_POSTGRES_DB}
+      POSTGRES_USER: ${IRR_POSTGRES_USER}
+      POSTGRES_PASSWORD: ${IRR_POSTGRES_PASSWORD}
+      PGPORT: ${IRR_POSTGRES_PORT}
+    healthcheck:
+      test: pg_isready -U ${IRR_POSTGRES_USER} -d ${IRR_POSTGRES_DB}
+      start_period: 10s
+      retries: 3
+      interval: 10s
+  irr_backend:
+    depends_on:
+      irr_db:
+        condition: service_healthy
```

Comment:
I believe you also need to add the gatekeeper as your dependency, since you need the gatekeeper to be up in order to register the service endpoints there